Bottom: a49b8d132b5f15902dc7ad1518793970f32e3fd8
Top:    098543f8afd604f23e885080b5eed39f1fcef222
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 17:26:26 +0000

Make --enable-dbus default


---

diff --git a/configure.ac b/configure.ac
index 4ab6e37..7a4cc76 100644
--- a/configure.ac
+++ b/configure.ac
@@ -327,7 +327,13 @@ fi
 AC_MSG_CHECKING([for whether to use DBUS])
 AC_ARG_ENABLE([dbus],
 [  --enable-dbus           Enable DBUS IPC],
-[],[enable_dbus=no])
+[],[
+	case " $with_gui " in
+		*\ gtk\ *)     enable_dbus=yes ;;
+		*\ lesstif\ *) enable_dbus=yes ;;
+		* )            enable_dbus=no ;;
+	esac
+])
 
 AC_MSG_RESULT([$enable_dbus])
 AM_CONDITIONAL(WITH_DBUS, test x$enable_dbus = xyes)
